Makefile

```diff
@@ -11,6 +11,11 @@ RENDER_COMMAND=$(VENVDIR)/bin/python3 build.py -j $(JOBS) -o $(OUTPUT_DIR)
 html: venv
 	$(RENDER_COMMAND)
 
+## htmlview       to open the index page built by the html target in your browser
+.PHONY: htmlview
+htmlview: html
```

Comment:
Or maybe just `view:`?

Reply:
`view` would be quite nice as in `make html view`, but we already have `make htmlview` in at least CPython docs, devguide, pythondotorg, so let's go for consistency.

Thanks!